addon/experiment-data.json

```diff
@@ -71,7 +71,6 @@
         "avatar": "\/\/www.gravatar.com\/avatar\/abe193c513e181a8a3ee40980928af39?s=64"
       }
     ],
-    "installations_url": "http:\/\/testpilot.dev:8000\/api\/experiments\/1\/installations\/",
```

Comment:
Not sure why all these URLs have have escaped slashes in the JSON, or if they need em, but that can be a separate bug/PR.